src/AutoMapper/Configuration/PathConfigurationExpression.cs

```diff
@@ -28,6 +28,11 @@ public void MapFrom<TSourceMember>(Expression<Func<TSource, TSourceMember>> sour
             MapFromUntyped(sourceExpression);
         }
 
+        public void Ignore()
+        {
+            PathMapActions.Add(pm => pm.Ignored = true);
```

Comment:
Yes, you would have to return null when CanResolve is false and filter for that.